### PR TITLE
First week projects

### PR DESCRIPTION
**Pull request by MichaelGarbuzov**

### CODE REVIEW

File: First-Html-page/personal-site-challenge.html [hunk 1]
```diff
@@ -1,29 +1,93 @@
...
+        <p class="intro">I am a Web Developer.And I ❤️ to code.</p>
+      </div>
+      <hr />
+      <div class="skilss">
+        <h2>My Skills.</h2>
+        <div class="skill-row">
+          <img
+            class="photo1"
+            src="https://media.giphy.com/media/ZVik7pBtu9dNS/giphy.gif"
+            alt=""
```
Comment (Tamir198) [inline First-Html-page/personal-site-challenge.html:43]: You should use the alt attribute

File: First-Html-page/personal-site-challenge.html [hunk 2]
```diff
@@ -1,29 +1,93 @@
...
+      <div class="contact-me">
+        <h2>Get In Touch</h2>
+        <h3>If you like coding like i do.</h3>
+        <p class="contact1">
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quis illum
+          autem harum molestiae necessitatibus impedit placeat, rem consectetur
+          provident. Ex laudantium odit fugit quo aliquam corporis nulla quaerat
+          quas incidunt?.
+        </p>
+        <a class="myButton" href="mailto:michaelgarbuzov1994@email.com"
```
Comment (Tamir198) [inline First-Html-page/personal-site-challenge.html:78]: Can you find a better name for this? myButton is not really standing for anything and its hard to understand what is should be doing.

File: Fylo-FronedMentor-Challenge/style.css [hunk 3]
```diff
@@ -0,0 +1,210 @@
+* {
+  font-family: raleway;
+}
+
+body {
+  background-color: hsl(229, 57%, 11%);
+  background-image: url(/Fylo-FronedMentor-Challenge/assets/bg-desktop.png);
+  background-repeat: no-repeat; /* wont duplicate the background */
```
Comment (Tamir198) [inline Fylo-FronedMentor-Challenge/style.css:8]: Do you think this comment is helpful? 

the `background-repeat: no-repeat` already explains that the background should not repeat itself

File: Fylo-FronedMentor-Challenge/style.css [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+}
+.bubble:after {
+  content: "";
+  position: absolute;
+  display: block;
+  width: 0;
+  z-index: 1;
+  border-style: solid;
+  border-width: 20px 20px 0 0;
+  border-color: #ffffff transparent transparent transparent;
```
Comment (Tamir198) [inline Fylo-FronedMentor-Challenge/style.css:169]: You can use `  border-color:  transparent ;` instead don't you think?

**Review by Tamir198 — COMMENTED** [review]
Is this project running on your computer? When I run it I only see the starting files of the project.

Some general things I saw in your code:

- Some of your naming need to be improved, for example, `photo1 ` if not ideal. I am having hard time to understand what it should be doing.
 
- If you already use the img tag why not add alt description to it?
- You can use CSS variables for repeating values in your CSS files

**Comment by MichaelGarbuzov** [comment]
> Is this project running on your computer? When I run it I only see the starting files of the project.
> 
> Some general things I saw in your code:
> 
> * Some of your naming need to be improved, for example, `photo1 ` if not ideal. I am having hard time to understand what it should be doing.
> * If you already use the img tag why not add alt description to it?
> * You can use CSS variables for repeating values in your CSS files

**Pull request closed without merging**